packages/common-all/src/store/ConfigStore.ts

```diff
@@ -0,0 +1,336 @@
+import { errAsync, okAsync, ResultAsync } from "neverthrow";
+import { DendronError, IDendronError } from "../error";
+import { URI, Utils } from "vscode-uri";
+import { ConfigReadOpts, IConfigStore } from "./IConfigStore";
+import { IFileStore } from "./IFileStore";
+import { CONSTANTS, StatusCodes } from "../constants";
+import { ConfigUtils, DeepPartial } from "../utils";
+import { DendronConfig, DendronConfigValue, DVault } from "../types";
+import { fromStr, toStr } from "../yaml";
```

Comment:
```suggestion
import * as YamlUtils from "../yaml";
```
This way the context of the functions is visible.